packages/wrangler/src/pages/publish.tsx

```diff
@@ -62,6 +63,12 @@ export function Options(yargs: Argv) {
 				type: "boolean",
 				description: "Skip asset caching which speeds up builds",
 			},
+			"bundle-worker": {
+				type: "boolean",
+				default: false,
```

Comment:
Given how bundling is needed for D1 to function, and soon Wasm which is being worked on by Greg - perhaps we should enable this by default instead?